api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
@samchrisinger, @brianjgeiger - I went ahead and did the work to remove the `logs/{log_id}/contributors` endpoint.  When I put it in in the first place, in CR it came up that this wasn't the best way to handle it.   If you think the endpoint should stay I will add it back in.

I moved the user information to params - just the information that we need for the front end - names, is_active attribute, and link for the user.   Logs are a weird endpoint, having to handle lots of different types of information.  In my opinion, it is okay to return the bare minimum information we need under params to formulate the logs.  For example, if a node has been deleted, we need to access the node's title for the logs because accessing the node in the API will return Gone.  So we stick the bare minimum: node title and id in the log params.